roles/confluent.kafka_broker/templates/kafka_jolokia.properties.j2

```diff
@@ -0,0 +1,13 @@
+port={{kafka_broker_jolokia_port}}
+host=0.0.0.0
+{% if kafka_broker_jolokia_ssl_enabled|bool %}
+keystore={{kafka_broker_keystore_path}}
+keystorePassword={{kafka_broker_keystore_storepass}}
```

Comment:
Is the requirement around this to use the same keystore/truststore as the broker or is it to be independently secured?